README.md

```diff
@@ -21,14 +21,4 @@ A shelf-reading (inventory check) process for FOLIO items.  Using a laptop, a ba
 
 ## Deployment
 
-- Clone this repo
-- Install (via pip): flask, pyopenssl, folioclient
-- Create and configure `config.properties` based on the example
-
-For test purposes only (uses a self-signed cert, NOT secure):
-
-```
-flask --app application/app run --cert=adhoc --host=0.0.0.0 -p some-port-number &
-```
-
-A production-safe Docker-based deployment is in the works.
+See https://github.com/lehigh-university-libraries/linderman?tab=readme-ov-file#initial-setup
```

Comment:
I think we should keep

> - Create and configure `config.properties` based on the example

since that's not covered by the general linderman documentation.

Also maybe leave the old instructions under a 'testing' header, and put this link under 'docker'?  Presuming that other institution users may need to find a docker alternative, they can't just go the test route but it may be a starting point?  Not sure.

Reply:
Oh yeah good call. I tried blending the two together https://github.com/lehigh-university-libraries/folio-offline-shelf-reading/blob/docker/README.md

Reply:
Looks good!